webextension/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
nit: we should probably move the webextension/.eslintrc.js and webextension/.stylelintrc files into the project root instead of the webextension/ directory so they don't get bundled into the add-on.

Reply:
+1